source/threads.tex

```diff
@@ -147,7 +147,7 @@
 An \defn{execution agent} is an entity such as a thread that may perform work in parallel with
 other execution agents. \begin{note} Implementations or users may introduce other kinds of
 agents such as processes or thread-pool tasks. \end{note} The calling agent is determined by
-context, e.g. the calling thread that contains the call, and so on.
+context, e.g., the calling thread that contains the call, and so on.
```

Comment:
I actually thing the entire "e.g." part needs a more drastic rewrite. This just doesn't read well.
And what "call" are we talking about? A "function call" in the syntactic sense of [expr.call]?
Suggestion: "is determined by the context, such as the thread of execution that contains the invocation of the function."  (This is consistent with [intro.multithread] p2.)

Oh, and does this note attempt to define the term "calling agent"? Then it would be totally misguided.